packages/expo-face-detector/android/src/main/java/expo/modules/facedetector/FaceDetectorUtils.kt

```diff
@@ -1,27 +1,34 @@
 package expo.modules.facedetector
 
+import android.graphics.PointF
 import android.os.Bundle
-import com.google.android.gms.vision.face.Landmark
-import com.google.firebase.ml.vision.face.FirebaseVisionFace
-import com.google.firebase.ml.vision.common.FirebaseVisionPoint
+import com.google.mlkit.vision.face.Face
+import com.google.mlkit.vision.face.FaceLandmark
 
 object FaceDetectorUtils {
   @JvmStatic
   @JvmOverloads
-  fun serializeFace(face: FirebaseVisionFace, scaleX: Double = 1.0, scaleY: Double = 1.0): Bundle {
+  fun serializeFace(face: Face, scaleX: Double = 1.0, scaleY: Double = 1.0): Bundle {
     val encodedFace = Bundle().apply {
-      putInt("faceID", face.trackingId)
+      face.trackingId?.let { putInt("faceID", it) }
       putDouble("rollAngle", face.headEulerAngleZ.toDouble())
       putDouble("yawAngle", face.headEulerAngleY.toDouble())
 
-      if (face.smilingProbability >= 0) {
-        putDouble("smilingProbability", face.smilingProbability.toDouble())
+      face.smilingProbability?.let {
+        if (it >= 0) {
+          putDouble("smilingProbability", it.toDouble())
+        }
       }
-      if (face.leftEyeOpenProbability >= 0) {
-        putDouble("leftEyeOpenProbability", face.leftEyeOpenProbability.toDouble())
+      face.smilingProbability?.let {
```

Comment:
```suggestion
      face.leftEyeOpenProbability?.let {
```
i'm not pretty sure but it looks like a little strange here.